include/CppUTestExt/MockSupport.h

```diff
@@ -135,7 +135,8 @@ class MockSupport
     MockExpectedCallComposite compositeCalls_;
     MockNamedValueComparatorsAndCopiersRepository comparatorsAndCopiersRepository_;
     MockNamedValueList data_;
-
+    SimpleString scope_;
```

Comment:
I guess you can't get the name from GetData, but it would probably be best to eliminate the duplication, or did you already?